### DISCUSSION TITLE
Does this support native backends for mac linux and windows?

### DISCUSSION THREAD

**Discussion by Shadowblitz16** [unoplatform/uno#4815] [Q&A] [answered]
Does this support native backends for mac linux and windows?
windows < 10 -> win32
windows >= 10 -> uwp
linux -> gtk
mac -> cocoa?

I want to develop native cross platform apps on desktops and phone with xaml.
I was pointed here from the winUI github.

I also want to support older versions of windows like windows 95

**Comment by davidjohnoliver** [comment, marked as answer]
> Does this support native backends for mac linux and windows?
> windows < 10 -> win32
> windows >= 10 -> uwp

Yes and yes - you can write both Win32 and UWP apps with [WinUI 3](https://platform.uno/docs/articles/get-started-dotnet-new.html#uno-platform-blank-application-for-winui-30---preview)

> linux -> gtk

Not exactly - Uno uses Gtk as a host on Linux, but mainly as an app shell and for handling input - the rendering is all done with Skia

> mac -> cocoa?

Yes, Uno for macOS uses Cocoa/AppKit as the graphical backend

> 
> I want to develop native cross platform apps on desktops and phone with xaml.
> I was pointed here from the winUI github.
> 

Welcome! Take a look [here](https://platform.uno/docs/articles/getting-started-tutorial-1.html) to get started :)

> I also want to support older versions of windows like windows 95

The oldest Windows version supported 'out of the box' by Uno is Windows 7. You might be able to rig up something to support older versions using WebAssembly and a PWA/Electron-like approach, but I don't know if anyone's specifically investigated targeting Windows 95.

**Reply by Shadowblitz16** [reply]
ok thankyou
